services/Vat.js

```diff
@@ -0,0 +1,15 @@
+import CRUDService from './crud';
+
+export default class Vat extends CRUDService {
+
+    constructor($http){
+        super($http);
+        this.endpoint = 'vat';
+    }
+
+    
+    items(){
+        return this.$http.get(this.apiURL + this.endpoint + '/'  ).then((items) => items.data);
+    }
+
```

Comment:
Remove all methods except `constructor` since they already exists in the `CRUDService` that this extends on.